openapi/publicapi_spec.json

```diff
@@ -869,6 +869,15 @@
           }
         }
       },
+      "AuthorisationSummary" : {
+        "type" : "object",
+        "description" : "A structure representing authorisation summary",
```

Comment:
Would be good to align description with https://github.com/alphagov/pay-tech-docs/pull/651/files